packages/1155-contracts/src/delegation/ZoraCreator1155PremintExecutorImpl.sol

```diff
@@ -90,30 +53,41 @@ contract ZoraCreator1155PremintExecutorImpl is
     /// @param signature Signature of the creator of the token, which must match the signer of the premint config, or have permission to create new tokens on the erc1155 contract if it's already been created
     /// @param quantityToMint How many tokens to mint to the executor of this transaction once the token is created
```

Comment:
can update "executor" to "mintRecipient"

Reply:
done